helm/bridge-operator-chart/templates/deployment.yaml

```diff
@@ -1,15 +1,15 @@
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: apps/v1
+kind: DaemonSet
 metadata:
   name: bridge-operator
   namespace: {{ .Values.namespace }}
   labels:
     app: bridge-operator
 spec:
-  replicas: 1
   revisionHistoryLimit: 3
-  strategy:
-    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: bridge-operator
```

Comment:
Didn't we have selector in the Deployment? Weird. How did it work?

Reply:
Don't know.  I also only copied it from some other operator.